ElfUtils/ElfFileTest.cpp

```diff
@@ -2,15 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include <absl/strings/str_format.h>
```

Comment:
Just out of curiosity, when do you choose <> vs ""?

Reply:
<> for stl/standard c/linux/windows headers, "" for the rest.